### ISSUE TITLE
Need a template for biochemical reactions

### ISSUE THREAD

**Issue by cjmyers** [drdozer/shortbol#62]
Need a way to express a biochemical reaction.

**Comment by drdozer** [comment]
Yes. Could you give an example? We need to add an appropriate template for this.

**Comment by cjmyers** [comment]
How about something like:

ME_R02446 : moduleDefinition
   functionalComponent : ME_C06561
   functionalComponent : ME_C00509
   functionalComponent : enzyme
   interaction = ME_C06561 <-- enzyme --> ME_C00509

This is meant to represent a reversible reaction catalyzed by "enzyme”.

If you cannot use symbols, then maybe:

interaction = ME_C06561 reversibleReaction(enzyme) ME_C00509

OR

interaction = ME_C06561 reversibleReaction ME_C00509 catalyzedBy enzyme

Should create, 

Interaction of type BiochemicalReaction
	Participation ME_C06561 with role Reactant
	Participation ME_C00509 with role Product
	Participation enzyme with role Modifier

AND

Interaction of type BiochemicalReaction
	Participation ME_C06561 with role Product
	Participation ME_C00509 with role Reactant
	Participation enzyme with role Modifier

Need to be able to express multiple Reactants, Products, and Modifiers, too.  Another example from the figure I sent would need 2 reactants and 3 products.  Unfortunately, there is no means to express stoichiometry in SBOL at this time, but maybe we can do this as an annotation.
